pam_yubico.c

```diff
@@ -398,8 +398,11 @@ authorize_user_token_ldap (struct cfg *cfg,
       goto done;
     }
 
+
   if (cfg->ldap_filter) {
-    filter = filter_printf(cfg->ldap_filter, user);
+    char* user_rfc4515 = rfc4515_replace( user );
```

Comment:
What do you think about including this in the filter_printf() function rather?

Reply:
Hey @klali, I don't see any reason why we can't integrate this into the filter_printf(...) function.

Reply:
done